week-2-practise/level-2/2-validation.js

```diff
@@ -1,11 +1,30 @@
+import { numberLiteralTypeAnnotation, returnStatement } from "@babel/types";
```

Comment:
You don't need this line, I don't think. At least your code runs correctly on my machine without it.